core/src/main/scala/kafka/admin/AclCommand.scala

```diff
@@ -62,7 +62,12 @@ object AclCommand {
     var authorizerProperties = Map.empty[String, Any]
     if (opts.options.has(opts.authorizerPropertiesOpt)) {
       val props = opts.options.valuesOf(opts.authorizerPropertiesOpt).asScala.map(_.split("="))
-      props.foreach(pair => authorizerProperties += (pair(0).trim -> pair(1).trim))
+      props.foreach {
+        case Array(key: String, value: String) => authorizerProperties += (key.trim -> value.trim)
+        case array =>
+          println(s"Invalid authorizer property ${array.mkString(",")}")
```

Comment:
Would it make sense to just throw an exception here and let the catch in main handle and exit?